script.js

```diff
@@ -11,4 +13,328 @@ hamburger.addEventListener('click', toggleMenu);
+
+  const popupTagsMob = document.createElement('ul');
+  popupTagsMob.classList.add('popup-items', 'mobile');
+  popupWindow.appendChild(popupTagsMob);
+
+  const popupTagsDesk = document.createElement('ul');
+  popupTagsDesk.classList.add('popup-items', 'desktop');
+  popupWindow.appendChild(popupTagsDesk);
+
+  for (let j = 0; j < project.languages.length; j += 1) {
+    const popupTag = document.createElement('li');
+    popupTag.innerHTML = project.languages[j];
+    popupTagsMob.appendChild(popupTag);
+  }
+  for (let j = 0; j < project.popupLanguages.length; j += 1) {
+    const popupTag = document.createElement('li');
+    popupTag.innerHTML = project.popupLanguages[j];
+    popupTagsDesk.appendChild(popupTag);
+  }
+
+  const popupDescription = document.createElement('p');
+  popupDescription.classList.add('popup-text');
+  popupDescription.innerHTML = project.description;
+  popupWindow.appendChild(popupDescription);
+
+  const popupButtonsContainerMobile = document.createElement('div');
+  popupButtonsContainerMobile.classList.add('popup-buttons', 'mobile');
+  popupWindow.appendChild(popupButtonsContainerMobile);
+
+  const popupLiveLink = document.createElement('button');
+  popupLiveLink.classList.add('button');
+  popupLiveLink.innerHTML = 'See Live';
+  popupButtonsContainerMobile.appendChild(popupLiveLink);
+
+  const popupLiveLinkIcon = document.createElement('img');
+  popupLiveLinkIcon.setAttribute('src', 'images/see-live-icon.png');
+  popupLiveLink.appendChild(popupLiveLinkIcon);
+
+  const popupSourceLink = document.createElement('button');
+  popupSourceLink.classList.add('button');
+  popupSourceLink.innerHTML = 'See Source';
+  popupButtonsContainerMobile.appendChild(popupSourceLink);
+
+  const popupSourceLinkIcon = document.createElement('img');
+  popupSourceLinkIcon.setAttribute(
+    'src',
+    'images/See-source.png',
+  );
+  popupSourceLinkIcon.setAttribute('alt', 'Source Link Icon');
+  popupSourceLink.appendChild(popupSourceLinkIcon);
+
+  x.appendChild(popupWindow);
+
+  document.body.appendChild(popupWindow);
+
+  popupCloseButton.addEventListener('click', () => {
+    overlay.style.display = 'none';
+    popupWindow.style.display = 'none';
+  });
+
+  popupLiveLink.addEventListener('click', () => {
+    window.open(project.liveLink);
+  });
+
+  popupSourceLink.addEventListener('click', () => {
+    window.open(project.sourceLink);
+  });
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  const projectButtons = document.querySelectorAll('#project-button');
+
+  for (let i = 0; i < projectButtons.length; i += 1) {
+    ((index) => {
+      projectButtons[index].addEventListener('click', () => {
+        createPopup(index);
+      });
+    })(i);
+  }
+});
+/*
+function validationForm() {
+  const email = document.querySelector('#mail');
+  const lowEmail = email.toLowerCase();
+
+  if (email === lowEmail) {
+    return true
+  }else {
+    alert()
+  }
+}
+*/
```

Comment:
👍 Thank you for making the necessary updates in response to the previous reviewer's feedback. However, I'd like to highlight a best practice concern regarding the commented-out code in your `script.js` file.

- [ ] As a recommended practice, it's better to completely remove code that is no longer needed, rather than commenting it out. Commented-out code can make the file less readable and cluttered, especially as your codebase grows. Removing unused code helps maintain a clean and organized codebase and ensures that future developers won't be confused by unnecessary or outdated code snippets.

By keeping your codebase clean and removing commented-out code, you contribute to better code readability and maintainability. Keep up the good work, and if you have any questions or need further assistance, feel free to ask! 🚀💻🌟